db/Pos/DB/GState/Common.hs

```diff
@@ -6,10 +6,9 @@ module Pos.DB.GState.Common
        (
          -- * Getters
          getTip
+       , getTipSomething
```

Comment:
1. You only use it in a single place.
2. I would prefer `getTipAbstract` (nitpicker-mode-on).

Reply:
I use it twice, in this file and in the `lib/DB/Block`